games-util/heroic-bin/heroic-bin-2.14.0.ebuild

```diff
@@ -5,7 +5,6 @@ EAPI=8
 
 APP_NAME="${P/-bin/}"
 APP_URI="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher"
-APP_RESOURCES_COMMIT=6dfb2758e531af693f0baffa15240f152aadd68b
```

Comment:
I check the url and the file is the same. It better to use the resources related to the $PV it install

Reply:
> I check the url and the file is the same. It better to use the resources related to the $PV it install

then on every pkgbump you will have the exact same duplicated X times in the Manifest file (X = number of heroic-bin ebuilds)

Reply:
Instead of commit hash you can pick a version Y and use that for every resource file. When files different just use a different version.

Reply:
> > I check the url and the file is the same. It better to use the resources related to the $PV it install
> 
> then on every pkgbump you will have the exact same duplicated X times in the Manifest file (X = number of heroic-bin ebuilds)

I never let the old Heroic version be around, since heroic team dosen't like bug report of old version. But if you prefer I can revert that change

Reply:
> I never let the old Heroic version be around, since heroic team dosen't like bug report of old version. But if you prefer I can revert that change

Sure, it's up to your judgement now and I respect that.